R/tune_bayes.R

```diff
@@ -632,7 +632,7 @@ fit_gp <- function(dat, pset, metric, eval_time = NULL, control, ...) {
 
 pred_gp <- function(object, pset, size = 5000, current = NULL, control) {
   pred_grid <-
-    dials::grid_latin_hypercube(pset, size = size) %>%
+    dials::grid_space_filling(pset, size = size, type = "latin_hypercube") %>%
```

Comment:
I stuck with a LHD here because the new designs take a really long time to compute a really large grid (5,000 points).